scripts/start

```diff
@@ -67,19 +67,20 @@ export DEVICE_HOSTNAME="${DEVICE_HOSTNAME}.local"
 export DOCKER_CLIENT_TIMEOUT=240
 export COMPOSE_HTTP_TIMEOUT=240
 
-cd "$UMBREL_ROOT"
 
 echo "Starting karen..."
 echo
-./karen &>> "${UMBREL_LOGS}/karen.log" &
+./karen
```

Comment:
I'm not really sure why you removed the `cd` command here. You could have just executed karen from the scripts dir:

```suggestion
cd "$UMBREL_ROOT"

echo "Starting karen..."
echo
./scripts/karen
```

Also, karen is already started by the system before the `start` script (as you specified in the `umbrel-startup`), so why do you manually execute it here? We can just check if it is Umbrel OS and start the karen service if it's not started (e.g. if a user manually use this script), and if it is not Umbrel OS, start karen manually. 

What do you think?

Reply:
> Also, on Umbrel OS karen is already started by the system before the start script (as you specified in the umbrel-startup), so why do you manually execute it here? 

karen itself already checks if it's running, so it just refuses to start and we dohn't have an additional check required here.

> I'm not really sure why you removed the cd command here. 

Because the two commands after the cd were run fromvthe script dir, only the two commands before were not, but I can revert that if you want.